log-configs/cluster.yaml

```diff
@@ -6,13 +6,11 @@ rotation:
 
 loggerTree:
   severity: Debug+
-  files:
```

Comment:
Does this means there will be no flat logs anymore? is that ok?

Reply:
There will be no more `node.log`, which is good. There are also flat logs written to stdout/stderr, which are captured in the systemd-journal.